sale_manual_delivery/models/stock_move.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    carrier_id = fields.Many2one("delivery.carrier", string="Delivery Method")
```

Comment:
@AaronHForgeFlow, why do we need to define the carrier at the stock.move level ?
cc @i-vyshnevska

Reply:
It is a requirement of the original authors of the module. https://github.com/OCA/sale-workflow/pull/404#discussion_r100574843

Reply:
@AaronHForgeFlow natively Odoo defines the carrier at stock picking level (in delivery module). It confuses us if we have multiple carriers in the same picking (stock.move level), but only one deliver provider defined in picking.

In actual usage, if a picking is delivered by different providers, the picking should be splitted (partial transfer + create back order).

Reply:
Splits are not performed anymore with this module. In this case two separate manual deliveries will be created.

Reply:
@i-vyshnevska, do you have any idea?